projects/storefrontstyles/scss/components/misc/popover/_popover.scss

```diff
@@ -0,0 +1,174 @@
+        border-width: $cx-popover-arrow-height ($cx-popover-arrow-width / 2) 0;
+        border-top-color: $cx-popover-border-color;
+      }
+      &::after {
+        bottom: $cx-popover-border-width;
+        border-width: $cx-popover-arrow-height ($cx-popover-arrow-width / 2) 0;
+        border-top-color: $cx-popover-arrow-color;
+      }
+    }
+  }
+
+  &.right,
+  &.right-top,
+  &.right-bottom {
+    margin-left: $cx-popover-arrow-height;
+    > .arrow {
+      left: subtract(-$cx-popover-arrow-height, $cx-popover-border-width);
+      width: $cx-popover-arrow-height;
+      height: $cx-popover-arrow-width;
+      margin: $cx-popover-border-radius 0;
+      &::before {
+        left: 0;
+        border-width: ($cx-popover-arrow-width / 2) $cx-popover-arrow-height
+          ($cx-popover-arrow-width / 2) 0;
+        border-right-color: $cx-popover-border-color;
+      }
+      &::after {
+        left: $cx-popover-border-width;
+        border-width: ($cx-popover-arrow-width / 2) $cx-popover-arrow-height
+          ($cx-popover-arrow-width / 2) 0;
+        border-right-color: $cx-popover-arrow-color;
+      }
+    }
+  }
+
+  &.left,
+  &.left-top,
+  &.left-bottom {
+    margin-right: $cx-popover-arrow-height;
+    > .arrow {
+      right: subtract(-$cx-popover-arrow-height, $cx-popover-border-width);
+      width: $cx-popover-arrow-height;
+      height: $cx-popover-arrow-width;
+      margin: $cx-popover-border-radius 0;
+      &::before {
+        right: 0;
+        border-width: ($cx-popover-arrow-width / 2) 0
+          ($cx-popover-arrow-width / 2) $cx-popover-arrow-height;
+        border-left-color: $cx-popover-border-color;
+      }
+      &::after {
+        right: $cx-popover-border-width;
+        border-width: ($cx-popover-arrow-width / 2) 0
+          ($cx-popover-arrow-width / 2) $cx-popover-arrow-height;
+        border-left-color: $cx-popover-arrow-color;
+      }
+    }
+  }
+
+  &.bottom,
+  &.bottom-left,
+  &.bottom-right {
+    margin-top: $cx-popover-arrow-height;
+    > .arrow {
+      top: subtract(-$cx-popover-arrow-height, $cx-popover-border-width);
+      &::before {
+        top: 0;
+        border-width: 0 ($cx-popover-arrow-width / 2) $cx-popover-arrow-height
+          ($cx-popover-arrow-width / 2);
+        border-bottom-color: $cx-popover-border-color;
+      }
+      &::after {
+        top: $cx-popover-border-width;
+        border-width: 0 ($cx-popover-arrow-width / 2) $cx-popover-arrow-height
+          ($cx-popover-arrow-width / 2);
+        border-bottom-color: $cx-popover-arrow-color;
+      }
+    }
+  }
+
+  &.top > .arrow,
+  &.bottom > .arrow {
+    left: 50%;
+    margin-left: -$cx-popover-arrow-width / 2;
+  }
+
+  &.top-left > .arrow,
+  &.bottom-left > .arrow {
+    left: 2em;
+  }
+
+  &.top-right > .arrow,
+  &.bottom-right > .arrow {
+    left: auto;
+    right: 2em;
+  }
+
+  &.left > .arrow,
+  &.right > .arrow {
+    top: 50%;
+    margin-top: -$cx-popover-arrow-width / 2;
```

Comment:
Any padding and margin-left/right should be changed to -inline-start and -inline-end. 
`(padding or margin)-inline-start` for left 
`(padding or margin)-inline-end` for right